docs/sni-limitation.rst

```diff
@@ -0,0 +1,35 @@
+.. _sni-limitation:
+
+Configure SNI TLS Load Balancer
+===============================
+
+Background
+----------
+
+Creating a new *TERMINATED_HTTPS* load balancer(LB for short) with new certificate(s) on OpenStack dashboard will trigger a new virtual server(VS for short) with new SSL profile(s) created on BIGIP side.
+
+It is possible to reuse existing certificates for creating new LB, which will trigger on BIGIP side, reusing existing SSL profiles for creating new VS.
+
+In summary:
+
+* Relation between certificate and SSL profile is one-to-one.
+* Certificate can associate to multiple LBs just as SSL profile can associate to multiple VSs on BIGIP side.
+
+
+SNI TLS Load Balancer
+---------------------
+
+The LB using multiple certificates called **SNI TLS Load Balancer**. Correspondingly, the triggered VS has multiple SSL profiles.
+
+Among SSL profiles of a specific LB, only one of them has attribute "*Default SSL Profile for SNI: Enabled*".
+
+**Using two or more SSL profiles of "Default SSL Profile for SNI: Enabled" will cause VS creation failure.** That is the limitation of using multiple SSL profiles.
+
+But unfortunately, it is hard for OpenStack dashboard to determine whether or not the SSL profile associated with a certificate does have "*Default SSL Profile for SNI: Enabled*" on BIGIP side, unless:
+
+* On BIGIP side, manually check that SSL profile's attribute, or
+* Name the certificate with special hint(i.e. suffix with _sni_default).
```

Comment:
This doesn't really make sense. How does including `_sni-default` in the certificate name in the dashboard ensure that that cert is marked as the default for SNI on the BIG-IP?

Reply:
We hope to suggest the user not to share certificates between SNI load balancer. For instance, if a user hope to create one SNI load balancer which include 3 certificates (A/B/C), those 3 certificates should only be associated with this load balancer. If that user hope to use cert A to create another load balancer, he need to create a new cert in Barbican with A's payload. Then, there will be a new cert in BigIP, and the new load balancer will not reuse the prior one.

Maybe Andrew did not describe the point very clear. Can you please help to re-phrase that? Thanks!